src/main/groovy/life/qbic/services/connectors/ConsulConnector.groovy

```diff
@@ -1,21 +1,24 @@
 package life.qbic.services.connectors
 
 import groovy.json.JsonSlurper
-import io.micronaut.http.client.RxHttpClient
-import io.micronaut.http.uri.UriBuilder
+
 import life.qbic.services.Service
 import life.qbic.services.ServiceConnector
 import life.qbic.services.ServiceType
 
+import java.net.http.HttpClient
+import java.net.http.HttpRequest
+import java.net.http.HttpResponse
+
```

Comment:
locally my maven cannot resolve this packages. Do they require java8?

Reply:
but this is definitely the part we need to downgrade, then it should work

Reply:
ok, how do you suggest to solve this?

Reply:
Java 11 is provided currently but still I cannot resolve the package

Reply:
ahh nvm.. my intellij project settings said java8.